addons/web/static/src/core/dropdown/dropdown.scss

```diff
@@ -15,7 +15,7 @@
   cursor: pointer;
 }
 
-button.dropdown-toggle {
+button.dropdown-toggle, a.dropdown-toggle {
```

Comment:
Was about to merge but in fact you are modifying a rule in `/web` and I am not sure how it is related to the commit's purpose, can you explain?

Reply:
The dropdowns in the pseudo-backend of website are using "a" instead of "button". They get rendered with an outline when they are opened without clicking on them first.
I will add the properties into the definition within website instead.

Reply:
> I will add the properties into the definition within website instead.

Ok :+1: 

Indeed what I did not get is how that file is used in website... but it is actually loaded for all visitors as well as long as many other unneeded stuff since I let that go through: https://github.com/odoo/odoo/commit/db7b485d984e50fae5a52308634166460e5fc908 Let's not reenforce those rules which do impact visitors websites :/

Hopefully we will get rid of this with our backend / frontend merge... :/

Reply:
Change done and tested on runbot.